server-common/src/main/java/org/apache/gravitino/server/web/Utils.java

```diff
@@ -48,6 +49,14 @@ public static <T> Response ok(T t) {
     return Response.status(Response.Status.OK).entity(t).type(MediaType.APPLICATION_JSON).build();
   }
 
+  public static Response created() {
+    return Response.status(Response.Status.CREATED).type(MediaType.APPLICATION_JSON).build();
+  }
```

Comment:
Shall we return OK or CREATED? In Gravitino, we don't use CREATED because it is a different HTTP code.

Reply:
Spark Lineage client will throw exception for HTTP code `OK`